lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
```

Comment:
What do you mean? It only binds the `this` value. And for the strictest compliance `method.length === 0` because the IDL specifies label as an optional parameter.